build/dockerfiles/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# syntax = docker/dockerfile:1.2
+# syntax = registry.erda.cloud/retag/dockerfile:1.2
```

Comment:
It seems feasible to upgrade to version 1.8.

Reply:
thanks, i set it to latest

`# syntax = registry.erda.cloud/retag/dockerfile:latest`